exporter/exporterparser/kafka.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package exporterparser
+
+import (
+	"context"
+	"fmt"
```

Comment:
Please separate stdlib imports from other imports i.e.
```go
import (
    "context"
    "fmt"

    "go.opencensus.io/trace"
    ...
)
```

Reply:
Got it, thanks.